eslint.config.js

```diff
@@ -0,0 +1,53 @@
+import js from "@eslint/js";
+import prettier from "./node_modules/eslint-config-prettier/prettier.js";
+import gts from "./node_modules/gts/build/src/index.js";
+
+
+export default [
+  js.configs.recommended,
+  prettier,
+  {
+    rules: gts.rules,
```

Comment:
Have you tried to see if these rules are working? I see these issues:

https://github.com/google/gts/issues/847#issuecomment-2075186393
https://github.com/google/gts/issues/830

I don't think anything is stopping us from using the flat config but I couldn't find any examples of gts being configured in the flat config format. So I figured I would ask.

Reply:
Yes, it seems to work.  E.g., it complains if I mess up the formatting or have any unused variable.